Partial backport of 0a410e76f551c6e6cf6d128f618208049d6a952c
(localedata: CLDRv29: update LC_TIME week/first_week,workday fields)
to fix just the es_CL locale.

Index: b/localedata/locales/es_CL
===================================================================
--- a/localedata/locales/es_CL
+++ b/localedata/locales/es_CL
@@ -118,6 +118,8 @@ t_fmt_ampm ""
 date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
+week 7;19971130;1
+first_weekday 2
 END LC_TIME
 
 LC_PAPER
